_posts/2015-04-28-feedhenry-push-templates.md

```diff
@@ -0,0 +1,70 @@
+
+For the clients that we want to use (e.g. Android iOS) we need to create variants, for more information on how to do this have a look at the UPS documentation for [Android](https://aerogear.org/docs/unifiedpush/aerogear-push-android/) and [iOS](https://aerogear.org/docs/unifiedpush/aerogear-push-ios/)
+
+Next we need to do two things we need create a "Service connector" for UPS and we need to tell the Cloud App the `GUID` of the "Service connector" so that it can use this to send messages to UPS. First we setup the "Service connector" to connect to our openshift UPS instance, go to the `Services & APIs` tab and click `Provision mBaaS Service/API`:
+
+![Provision mBaaS Service](/img/news/2015-04-20-feedhenry-push-templates/3.png)
+
+Then choose `AeroGear Push Service`, give it a name and click `Next`, fill in the `Server URL`, `Application ID` and `Master Secret` and click `Next` now it will create your "Service connector"
+
+![create service connector](/img/news/2015-04-20-feedhenry-push-templates/7.png)
+ 
+ After that is done go back to your project, here we need to add the service so that we can use it click the "+" in the mBaas Services box in the overview page:
+
+![add mBaas Services](/img/news/2015-04-20-feedhenry-push-templates/2.png)
+
+Select the created "Service connector" and then the "Associate Service" button. Now you'll see the "Service Connector" in the "mBaas Services" column. Let's configure the "Cloud App" to use this "Service Connector", for that we need to the GUID of the "Service Connector", click on it and you will see the info page:
+
+![copy GUID](/img/news/2015-04-20-feedhenry-push-templates/4.png)
+
+On the info page copy the `Service ID` and under access control add you project, then click save.
+
+Then in the project click on the "Push Cloud App" and select "Environment variables" from the sidebar. Click "Add Variable" and enter `AEROGEAR_SERVICE_GUID` for name and paste the `Service ID` you copied earlier for the "Development" label. Next click on "Push Environment Variables" to publish these.
+
+![environment variable](/img/news/2015-04-20-feedhenry-push-templates/6.png)
+
+Now that that is all setup, the "Cloud App" can use the "GUID" to fetch the "Service Connector", that connects to UPS to send the messages.
+
+Let's fire up a browser and open the _Push Console App_ to create a couple of categories for instance: Football, Rugby and Basketball: 
+
+![console](/img/news/2015-04-20-feedhenry-push-templates/8.png)
+
+Now go to your _Push Notifications Mobile Client_ and open the file called `push-config.json`. You'll need to update the _Variant ID_ and _Variant Secret_ for each platform, so that it can register itself with the UnifiedPush Server in order to receive messages. For Android you also need to specify the _projectID_ inside of that JSON file.  Finally build and install the _Push Notifications Mobile Client_ on a device and select the categories you want to receive.
```

Comment:
perhaps we can donate a sentence or two to minimize confusion on the
differnt terminology here

On Wed, Apr 29, 2015 at 3:06 PM, Erik Jan de Wit <notifications@github.com>
wrote:

> In _posts/2015-04-28-feedhenry-push-templates.md
> <https://github.com/aerogear/aerogear.org/pull/581#discussion_r29333118>:
>
> > +
> > +![copy GUID](/img/news/2015-04-20-feedhenry-push-templates/4.png)
> > +
> > +On the info page copy the `Service ID` and under access control add you project, then click save.
> > +
> > +Then in the project click on the "Push Cloud App" and select "Environment variables" from the sidebar. Click "Add Variable" and enter `AEROGEAR_SERVICE_GUID` for name and paste the `Service ID` you copied earlier for the "Development" label. Next click on "Push Environment Variables" to publish these.
> > +
> > +![environment variable](/img/news/2015-04-20-feedhenry-push-templates/6.png)
> > +
> > +Now that that is all setup, the "Cloud App" can use the "GUID" to fetch the "Service Connector", that connects to UPS to send the messages.
> > +
> > +Let's fire up a browser and open the _Push Console App_ to create a couple of categories for instance: Football, Rugby and Basketball:
> > +
> > +![console](/img/news/2015-04-20-feedhenry-push-templates/8.png)
> > +
> > +Now go to your _Push Notifications Mobile Client_ and open the file called `push-config.json`. You'll need to update the _Variant ID_ and _Variant Secret_ for each platform, so that it can register itself with the UnifiedPush Server in order to receive messages. For Android you also need to specify the _projectID_ inside of that JSON file.  Finally build and install the _Push Notifications Mobile Client_ on a device and select the categories you want to receive.
>
> No way to change this 'Project Id' named field as it has different names
> on the google development console as well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear.org/pull/581/files#r29333118>.
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf